src/ui/summary_page.rs

```diff
@@ -478,12 +482,73 @@ impl SummaryPage {
             .style(ratatui::style::Style::default().fg(ratatui::style::Color::White));
         frame.render_widget(attest_status_widget, status_rect);
     }
+
+    fn render_interface_list(&self, model: &Rc<Model>, list_rect: Rect, frame: &mut Frame) {
+        // create header for the table
+        let header = Row::new(vec![
+            Cell::from("Name").style(Style::default()),
+            Cell::from("Link").style(Style::default()),
+            Cell::from("IPv4/IPv6").style(Style::default()),
+            Cell::from("MAC").style(Style::default()),
+            Cell::from("Controller").style(Style::default()),
+        ]);
+
+        // create list items from the interface
+        let rows = model
+            .borrow()
+            .network
+            .iter()
+            .map(|iface| super::networkpage::info_row_from_iface(iface))
+            .collect::<Vec<_>>();
+
+        // create a surrounding block for the list
+        let block = Block::default()
+            .title(" Network Interfaces ")
+            .title_alignment(Alignment::Center)
+            .borders(Borders::ALL)
+            .border_type(BorderType::Plain)
+            // .border_style(Style::default().fg(Color::White).bg(Color::Black))
```

Comment:
Remove unused code?

Reply:
@uncleDecart this comented on purpose. when 16 color support is implemented I'll uncomment